packages/cli/lib/polling.js

```diff
@@ -0,0 +1,22 @@
+const { POLLING_DELAY } = require('./constants');
+
+const poll = (callback, accountId, taskId) => {
+  return new Promise((resolve, reject) => {
+    const pollInterval = setInterval(async () => {
+      const pollResp = await callback(accountId, taskId);
+      const { status } = pollResp;
+
+      if (status === 'SUCCESS') {
```

Comment:
Might be worth making a constant for these statuses

Reply:
Addressed in https://github.com/HubSpot/hubspot-cli/pull/1052/commits/d26a4b34b0899f3de6e354bfccf416f97b3278ec